gninasrc/lib/model.h

```diff
@@ -255,10 +255,16 @@ struct ligand : public flexible_body, atom_range {
 
 struct residue : public main_branch {
     residue() {
-    } //serialization
+    } // for serialization
     residue(const main_branch& m)
         : main_branch(m) {
     }
+
+    friend class boost::serialization::access;
+    template<class Archive>
+    void serialize(Archive& ar, const unsigned version) {
+      ar & boost::serialization::base_object<main_branch>(*this);
+    }
 };

```

Comment:
adding this is fine, but don't like mixing these unrelated changes into a pull request for something else

Reply:
Ok, I can split this change on a different PR.